packages/bee-q/src/components/breadcrumb/_storybook/bq-breadcrumb.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import type { Args, Meta, StoryObj } from '@storybook/web-components';
+
+import { html } from 'lit-html';
+
+import mdx from './bq-breadcrumb.mdx';
+
+const meta: Meta = {
+  title: 'Components/Breadcrumb',
+  component: 'bq-breadcrumb',
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+  argTypes: {
+    text: { control: 'text', table: { disable: true } },
+    ['separator-icon']: { control: 'text' },
+  },
+  args: {
+    text: 'text',
+    ['separator-icon']: '',
+  },
+};
+export default meta;
+
+type Story = StoryObj;
+
+const Template = (args: Args) => html`
+  <bq-breadcrumb separator-icon=${args['separator-icon']}>
+    <bq-breadcrumb-item>
+      <div class="flex items-center gap-xs">
+        <bq-icon name="house-line" size="16"></bq-icon>
+      </div>
+    </bq-breadcrumb-item>
+    <bq-breadcrumb-item>Men's Clothing</bq-breadcrumb-item>
+    <bq-breadcrumb-item>
+      <div class="flex items-center gap-xs">
+        <bq-icon name="shirt-folded" size="16"></bq-icon>
+        Shirts
+      </div>
+    </bq-breadcrumb-item>
```

Comment:
Actually, with one single slot, we can still guarantee the layout, my point is to have consumers do something like:

```html
    <bq-breadcrumb-item>
      <bq-icon name="shirt-folded" size="16"></bq-icon>
      Shirts
    </bq-breadcrumb-item>
```